qemu/overlay/bin/monitord

```diff
@@ -0,0 +1 @@
+../../../bin/monitord
```

Comment:
this doesn't exist anymore

Reply:
Again, very good catch, man you are on fire :fire: